webapp/channels/src/components/channel_notifications_modal/components/notification_section.test.tsx

```diff
@@ -31,7 +31,7 @@ describe('components/channel_notifications_modal/NotificationSection', () => {
     });
 
     test('should match snapshot, DESKTOP on expanded view', () => {
-        const props = {...baseProps, expand: true};
+        const props= {...baseProps, expand: true};
```

Comment:
nit: I would keep the space before equal sign as most probably style check will complain about it.

Reply:
Thanks @sinansonmez, I added 2 more commits for typos and precaution for style check.

Reply:
Didn't see your approvement before making changes, sorry  :(

Reply:
No worries at all. However, in this file, I was trying to say we should keep the space before equal like `const wrapper = shallow`. I have seen that you have removed all spaces before equal sign. Can you revert that? Sorry for the confusion

Reply:
Haha, my bad.  on it.